### ISSUE TITLE
这样改是不是好玩一点(未测试，4gb显存估计测不了一点)

### ISSUE THREAD

**Issue by zmwv823** [smthemex/ComfyUI_DiffuEraser#10]
### 这是定制的brushnet(看结构貌似controlnet变种)和modified unet，最终pipeline只缺少te、scheduler、feature_extractor和vae，所以不从ckpt初始化完整模型:
- 使用Comfy原生text_encode输出的conditioning作输入，这里填写提示词能否绘制需要的内容而不是擦除(估计能绘制也没用，一致性大概率没法保证。如果提示词没什么作用，不如直接保存c为safetensors文件，clip直接省了，反正都是encode空文本---需要验证是否对结果有影响)？
```
c, uc = positive[0][0], negative[0][0] #使用原生输入时，clip占用显存需要从原生k采样器释放，所以这里需要使用代码手动释放。
```
- 使用dict(config.json)初始化单个diffusers vae。
```
vae = AutoencoderKL(**config)
state_dict = load_torch_file(vae_path, safe_load=True)
vae.load_state_dict(state_dict, strict=False)
del state_dict #删除sd，释放内存
vae.to(vae_dtype)
```
- pipe = StableDiffusionDiffuEraserPipeline(vae, unet, scheduler, brushnet, feature_extractor) #在pipeline注释掉不需要的输入，或者改为预定义=None。

**Comment by smthemex** [comment]
我测试过单独剔出clip和vae， DiffuEraser类跑图有问题，目前还在解决，实际剔除clip和vae后，显存峰值5G多点
